src/Runner.Listener/BrokerMessageListener.cs

```diff
@@ -329,7 +330,7 @@ public async Task<TaskAgentMessage> GetNextMessageAsync(CancellationToken token)
             }
         }
 
-        public async Task RefreshListenerTokenAsync(CancellationToken cancellationToken)
```

Comment:
removed the `CancellationToken cancellationToken` from the interface method since no one is using it.